manage_breast_screening/nhsuk_forms/fields/integer_field.py

```diff
@@ -33,3 +35,34 @@ def widget_attrs(self, widget):
         attrs.pop("step", None)
 
         return attrs
+
+
+class YearField(IntegerField):
+    def __init__(
+        self,
+        *args,
+        hint=None,
+        label_classes=None,
+        classes=None,
+        min_value=None,
+        max_value=None,
+        **kwargs,
+    ):
+        if min_value is None:
+            min_value = date.today().year - 80
```

Comment:
Just noticed a possible problem here - fields can be created when the form is declared, rather than when the form is instantiated, so the field's validation could get stale if the app doesn't restart. Maybe less of an issue here, but definitely in the SplitDateField.

Is it possible to make min_value/max_value a callable?